src/components/Auth/Verify.tsx

```diff
@@ -0,0 +1,124 @@
+    })
+  }
+
+  return (
+    <Flex direction='column'>
+      <Box me='auto'>
+        <Heading color={textColor} fontSize='36px' mb='10px'>
+          {title}
+        </Heading>
+        <Text mb='36px' ms='4px' color={textColorSecondary} fontWeight='400' fontSize='md'>
+          {subTitle}
+        </Text>
+      </Box>
+      {isLoading && !isError && (
+        <Box height={'100px'}>
+          <Loading minHeight={1} />
+        </Box>
+      )}
+      <Box>
+        <FormControl isInvalid={isError}>
+          {isError && (
+            <FormErrorMessage>
+              {error?.message || t('error.error_doing_things', { defaultValue: 'Error al realizar la operación' })}
+            </FormErrorMessage>
+          )}
+        </FormControl>
+      </Box>
+    </Flex>
+  )
+}
+
+export const VerifyAccountNeeded = ({ email }: { email: string }) => {
+  const { textColor, textColorSecondary } = useDarkMode()
+  const { t } = useTranslation()
+  const [emailSent, setEmailSent] = useState(false)
+  const { mutate, isError, error, isPending } = useResendVerificationMail({
+    onSuccess: () => {
+      setEmailSent(true)
+    },
+  })
+
+  const resendMail = useCallback(() => {
+    if (email && !emailSent) {
+      mutate({ email })
+    }
+  }, [emailSent, email])
+
+  return (
+    <Flex direction='column'>
+      <Box me='auto'>
+        <Heading color={textColor} fontSize='36px' mb='10px'>
+          {t('verify.account_created_succesfully', { defaultValue: 'Account created successfully!' })}
+        </Heading>
+        <Text mb='36px' ms='4px' color={textColorSecondary} fontWeight='400' fontSize='md'>
+          {t('verify.verification_email_is_sent', {
+            defaultValue: 'A verification email has been sent to:',
+          })}
+        </Text>
+        <Text mb='36px' ms='4px' color={textColorSecondary} fontWeight='bold' fontSize='md'>
+          {email}
+        </Text>
+        <Text mb='36px' ms='4px' color={textColorSecondary} fontWeight='400' fontSize='md'>
+          {t('verify.follow_email_instructions', {
+            defaultValue: 'Follow the instructions there to activate your account.',
+          })}
+        </Text>
+      </Box>
+      <Button isLoading={isPending} onClick={resendMail}>
+        <Trans i18nKey={'verify.resend_confirmation_mail'}>Resend Email</Trans>
+      </Button>
+      <Box pt={2}>
+        {emailSent && <Trans i18nKey={'verify.email_sent'}>Email sent successfully</Trans>}
+        <FormControl isInvalid={isError}>
+          {isError && <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>}
+        </FormControl>
+      </Box>
+      {import.meta.env.VOCDONI_ENVIRONMENT === 'dev' && (
+        <Button mt={4} as={ReactRouterLink} to={`/account/verify?email=${email}&code=`}>
+          Mail verification for dev envs
+        </Button>
+      )}
+    </Flex>
+  )
+}
+
+export default Verify
```

Comment:
@selankon I believe that this default export is unecessary. Can you please remove it and also update the import in the  router?

Reply:
We need default import to lazy load the page on router 

https://github.com/vocdoni/ui-scaffold/blob/53ee7cf570b909704e6352bc9f134c6e37fdcf10/src/router/Router.tsx#L37